tools/fw-logger/rs-fw-logger.cpp

```diff
@@ -1,17 +1,16 @@
 // License: Apache 2.0. See LICENSE file in root directory.
 // Copyright(c) 2015 Intel Corporation. All Rights Reserved.
 
-#include <librealsense2/rs.hpp>
+#include "librealsense2\rs.hpp"
+#include "librealsense2\hpp\rs_internal.hpp"
```

Comment:
You cannot use forward slash on anything but windows

Reply:
done